Add link to snapshot on readme and add back some gitignores for IDE junit tests

hazendaz opened the pull request:
Some users have asked for latest so makes sense to make it easy to get to by adding a direct link.  This is done in generic way so we don't need to keep changing it later.

Upon further testing within eclipse, it became obvious that without setting up properties for derby, the root of the project is still used.  So added back gitignores with note as to why.

emacarron commented:
Seems that I added and removed the "in progress" label while I was sleeping!! That must be a bug in github or sleepwalking! Hope it is the first ;)

BTW, good change!

hazendaz commented:
I thought you figured out auto tagging!


Thanks on the fix work.  I was trying to do something bigger.  I updated mockito to latest perpetual beta.  That resulted in on two test issues but fix is kind of ugly.  I also brought in assertj to replace hamcrest and junit assert that calls.  That was fairly clean and significantly more fluent.  I had to bring in one other library to deal with junit rules.  In all about 17 test classes changed.  Not sure if you will want all that.  While its better it doesnt prevent continuing to use more legacy style but might get people thinking since hamcrest is all but dead and junit 5 doesnt use it and if i recall junit rules may be out too.  I wouldnt have needed the additional library for rules if we started to run unit tests on java 7.  Its totally possible to run source targeting 6 while 7 on unit tests but depending on underlying usage eclipse can get confused.


Sent from Outlook Mobile






On Thu, Apr 28, 2016 at 9:23 PM -0700, "Eduardo Macarron" <notifications@github.com> wrote:





Seems that I added and removed the "in progress" label while I was sleeping!! That must be a bug in github or sleepwalking! Hope it is the first ;)

BTW, good change!

---
You are receiving this because you modified the open/close state.
Reply to this email directly or view it on GitHub:
https://github.com/mybatis/mybatis-3/pull/680#issuecomment-215623468
